test/common.js

```diff
@@ -598,8 +598,7 @@ exports.WPT = {
     try {
       fn();
     } catch (err) {
-      if (err instanceof Error)
-        err.message = `In ${desc}:\n  ${err.message}`;
+      console.error(`In ${desc}:`);
```

Comment:
No, it was intentional, since it seems like after the `Error` is constructed changing the `message` doesn't change `stack`. (See 56f6b8dbad36a46f07668b2b35574dc367e6d146.)